packages/now-build-utils/src/fs/node-version.ts

```diff
@@ -13,19 +14,19 @@ const allOptions: NodeVersion[] = [
     runtime: 'nodejs8.10',
     discontinueDate: new Date('2020-01-06'),
   },
-];
+] as const;
 
 const pleaseSet =
   'Please set "engines": { "node": "' +
   getLatestNodeVersion().range +
-  '" } in your `package.json` file to upgrade to Node.js ' +
+  '" } in your `package.json` file to use Node.js ' +
   getLatestNodeVersion().major +
   '.';
 const upstreamProvider =
   'This change is the result of a decision made by an upstream infrastructure provider (AWS).' +
   '\nRead more: https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html';
 
-export function getLatestNodeVersion(): NodeVersion {
+export function getLatestNodeVersion() {
```

Comment:
I removed `NodeVersion` here because I want the `as const` option returned.

This means `getLatestNodeVersion().range` will have the type `14.x` instead of `string` which will help surface other bugs downstream.